hudi-spark-datasource/hudi-spark2/src/main/antlr4/imports/SqlBase.g4

```diff
@@ -0,0 +1,1099 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This file is an adaptation of Presto's presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4 grammar.
```

Comment:
I thought we planned to just use Spark SQL keywords and limit to Spark3 (which should already recognize delete/merge?). Is antlr really needed for the first pass?

Reply:
i agree with pengzhiwei2018 that introduce antlr4 for spark2.   spark2 cannot use spark3‘ parser directly (many grammer is changed in spark3)

Reply:
Hi @vinothchandar , currently the sql parser is only for the spark2, I add the sql parser in the `hudi-spark2 ` project only. For spark3, as you have mentioned, it has support the delete/merge syntax, we do not need to extend the sql parser.